lib/ansible/module_utils/network/asa/asa.py

```diff
@@ -130,7 +130,7 @@ def run_commands(module, commands, check_rc=True):
 def get_config(module, flags=None):
     flags = [] if flags is None else flags
 
-    passwords = module.params['passwords']
+    passwords = module.params.get('passwords', False)
```

Comment:
You should absolutely put danger comments here to explain why you need to use `get()`